README.md

```diff
@@ -34,13 +34,14 @@ Alternativly, simply add these lines to your /etc/hosts:
 
 3. `npm install`
 
-4. `npm run start` to run against stage or prod environments. <br/>
-    OR <br/>
-  `npm run local` to run against a local backend running on port 8000.
+4. `npm run build`
```

Comment:
i would mention that this step is only necessary if the app isn't built yet (ie you don't have a /dist directory)